components/ui/AppCard.vue

```diff
@@ -17,12 +17,20 @@
         <h4 class="app-card__title">
           {{ title }}
         </h4>
-        <div v-if="hasTags" class="app-card__tags">
+        <div v-if="tags && Array.isArray(tags)" class="app-card__tags">
           <cv-tag
             v-for="tag in tags"
             :key="tag"
             :label="tag"
-            kind="purple"
+            kind="dark-purple"
+          />
+        </div>
+        <div v-else-if="tags" class="app-card__tags">
+          <cv-tag
+            v-for="tag in Object.values(tags).flat()"
+            :key="tag"
+            :label="tag"
+            :kind="tag === tags.tier ? 'purple' : 'dark-purple'"
```

Comment:
@javabster - please check the [Carbon Vue tag component](https://vue.carbondesignsystem.com/?path=/info/components-cvtag--standard), to see if this is allowed... I think `purple` is a valid value, however, I suspect that `dark-purple` is causing issues in the console:

![Screen Shot 2022-06-14 at 11 09 37 AM (2)](https://user-images.githubusercontent.com/6276074/173659543-87347cdc-8342-4919-a026-bd2bb8f751bc.png)


we might have to leverage one of the pre-existing colors?